int32 aruco_id
---
bool success
string message
